### PR TITLE
Post announcing the extension

### PR DESCRIPTION
**Pull request by erickt**
* Are there any other outstanding questions?
* Is there anything else we should setup before the announcement?
* Can you all help finish [Resisting Underhandedness](https://github.com/rust-community/underhanded.rs/wiki/Resisting-Underhandedness)?

### CODE REVIEW

**Comment by erickt** [comment]
[Rendered](https://github.com/erickt/underhanded.rs/blob/extend/_posts/2017-02-26-extending-submission-deadline.en-US.md)

File: _posts/2017-02-26-extending-submission-deadline.en-US.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Hello Underhanded Rustlers!  The [Rust Community Team][rust_community_team] has
+extended the submission deadline for the Underhanded Rust Contest to March 31,
+2017.  We wanted to help give people new to Rust a little more time to get
+ramped up on the language.  As part of this, we have developed a few new
+resources to get you on your way.
+
+First, we have created a simple example we're calling [quad-bank][quad_bank].
+It builds out a simple payment processor using a number of popular Rust
+libraries, like the SQL ORM [Diesel][diesel], and the web frameworks
+[Iron][iron] and [Rocket][rocket]. It also has contains what's probably the
```
Comment (Manishearth) [inline _posts/2017-02-26-extending-submission-deadline.en-US.md]: Mention that you can feel free to lift code from here?

**Review by Manishearth — APPROVED** [review]

File: _posts/2017-02-26-extending-submission-deadline.en-US.md (hunk 1 above)
Comment (erickt) [reply _posts/2017-02-26-extending-submission-deadline.en-US.md]: Good idea. I added that in the latest draft.

**Comment by frewsxcv** [comment]
Is it worth mentioning https://github.com/rust-fuzz/cargo-fuzz now that it's a thing?

**Comment by frewsxcv** [comment]
I guess it's not directly related to underhanded.rs, so nevermind.

**Comment by erickt** [comment]
@frewsxcv I added it to the [wiki page](https://github.com/rust-community/underhanded.rs/wiki/Resisting-Underhandedness). Feel free to add anything else to it, or change anything I wrote :)

**Pull request merged**